javascript/basic-js-practice.md

```diff
@@ -178,3 +178,17 @@ common([1, 2, 3], [1, 3]) → True
 ### [Challenge 10] Determine Runtime Complexity
 
 At the top your document for each challenge, state the runtime complexity for that function.
+
+
+### Check for Understanding
+
+- How create a function in Javascript?
+
+- Create 3 examples using Math()
+
+- How to return a function
```

Comment:
What do you mean here? ‘How does a function return its output’? Or ‘How do you call a function’?

Reply:
I think in "How does a function return its output" , propose an exercise to fix how to "return" the function

Reply:
ok, can you ask them to demonstrate it?